docs/.vuepress/config.js

```diff
@@ -89,9 +89,9 @@ module.exports = {
               title: 'Contribute',
               collapsable: false,
               children: [
-                ['contribute/how-to-contribute', 'How To Contribute'],
-                ['contribute/development-process', 'Development Process'],
-                ['contribute/bug-report-guidelines', 'Bug Report Guidelines']
+                ['contribute/bug-report-guidelines', 'Bug Report'],
+                ['contribute/how-to-contribute', 'Join Weex community'],
+                ['contribute/contribute-code', 'Contribute Code'],
```

Comment:
This changes the URL of that page. Is there a redirect of the old URL to the new one for Google and Co?

Reply:
1.  `development-process.html` has been renamed to `contribute-code.html`, which is the only URL change here.
1. As this change is happened on **draft** branch and the corresponding website is not launched officially, renaming file would be a issue in this case.